composer.json

```diff
@@ -34,6 +34,7 @@
 		"test": "phpunit"
 	},
 	"require-dev": {
-		"scrutinizer/ocular": "^1.3"
+		"scrutinizer/ocular": "^1.3",
+		"phpunit/phpunit": "^5.2"
```

Comment:
Yes, I can live with that. Just note that you should also change the Travis file because building will now fail for older PHP versions (because the dependency cannot be downloaded).